### DISCUSSION TITLE
Welcome to Csapat_3 Discussions!

### DISCUSSION THREAD

**Discussion by aalex94** [aalex94/Csapat_3#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fattila1** [comment]
Sziasztok! Még szokom a rendszert, úgyhogy ide is kommentelek. Szóval a feladat megjött, és betettem issue-ként. Látjátok?
